solutions/cf-1365D.mdx

```diff
@@ -230,7 +231,66 @@ public class SolveTheMaze {
 	//CodeSnip{Kattio}
 }
 ```
-	
+
 </JavaSection>
 
+<PySection>
+
+```py
+t = int(input())
+for _ in range(t):
+	n, m = map(int, input().split())
+
+	# Stores coordinates of all good people, which we will remove as we protect.
+	good_people = set()
+	grid = []
+	for r in range(n):
+		grid.append(list(input().strip()))
+		for c, char in enumerate(grid[-1]):
+			if char == 'G':
+				good_people.add((r, c))
+
+	visited = [[False] * len(grid[0]) for _ in range(len(grid))]
+
+	# Padding opimization
```

Comment:
what's this

Reply:
are you talking about visited or padding optimization?

Reply:
Part of the padding thing I also did a pull request for. It makes coding it easier since you don't have to type a bunch of inequalities in your floodfill loop. Should I do normal floodfill instead?

Reply:
i'd uh do normal floodfill tbh